File: Femis Assignments/Morning Snacks/GCD.py
first_number = int(input("Enter first number: "))
second_number = int(input("Enter second number: "))

gcd= 1

if first_number < second_number:
    smallest_number = first_number
else:
    smallest_number = second_number

for factor in range(1,smallest_number +1):
    if first_number % factor == 0 and second_number % factor == 0:
        gcd = factor
print(gcd)

File: Femis Assignments/Morning Snacks/GCD.js
const prompt = require('prompt-sync')();
let firstnumber = prompt("Enter first number: ");
let secondnumber = prompt("Enter second number: ");

let gcd = 1;
if (firstnumber < secondnumber){
    smallestnumber = firstnumber
}
else {
    smallestnumber = secondnumber
}
for (let factor = 1; factor <= smallestnumber; factor++){
 if (firstnumber % factor == 0 && secondnumber % factor == 0){
        gcd = factor
}
}
console.log(gcd)
